### PR TITLE
Create a code-review section

### PR DESCRIPTION
**Pull request by bobek**
Documenting the key concepts and requirements around code-review.

Also fixes PLOW-18

### CODE REVIEW

**Comment by bobek** [comment]
I am curious, if Exalate is capable of parsing ticket numbers from comments. This PR is fixing PLOW-18.

**Comment by bobek** [comment]
> I am curious, if Exalate is capable of parsing ticket numbers from comments. This PR is fixing PLOW-18.

Answer is no :(

**Review by AmitKumarDas — APPROVED** [review]
This is great. I was wondering if we do have some set of practices or must haves even before a project starts to have the business logic. For example, project should explain the folder structure, should have sample test code, sample scalability test code, automated doc gen, automated CI CD pipelines e.g. the simplest ones that can run in a single GitHub VM (_free of cost_).

**Review by arne-rusek — APPROVED** [review]
Few little suggestions. Very nice!

**Review by kmova — APPROVED** [review]

File: craft/code-review.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+Note: Google's [How to do a code review](https://google.github.io/eng-practices/review/reviewer/) heavily influenced this document.
+
+Code reviews are an essential part of our constant effort to improve quality of our code base. It is essential for keeping technical debt under control. Good code-review is a balancing act. We don't want to allow smelly code, but also going through review should not be a painful process, which will prevent developers from submitting features or improvements.
+
+Key attributes which reviewers are looking for
+
+- proposed change is consistent with the rest of the code-base.
+- proposed change is maintainable (or is not making existing code harder to maintain).
+- proposed change is improving the overall code health (even if the change itself is not perfect).
+- proposed change is addressing product/business need described in the parent ticket.
```
[the remark this answers is not in the document]
Comment (bobek) [reply craft/code-review.md:12]: This is always the hard one, correct solution would be to use `;`.

**Review by bobek — COMMENTED** [review]
Thank you for your suggestions!

**Pull request merged**